test/resources/test_maskinporten.py

```diff
@@ -904,7 +907,7 @@ def test_create_client_key_invalid_client_id(mock_authorizer, mock_client):
     assert res.status_code == 400
     assert (
         res.json()["message"]
-        == 'Invalid client ID (string does not match regex "^[0-9a-f-]+$")'
+        == "path.client_id: Invalid client ID (String should match pattern '^[0-9a-f-]+$')"
```

Comment:
Hm, den `path`-greia virka litt kryptisk. Er ikke det en internt Pydantic/FastAPI-greie?

Reply:
Er `path`, `query` eller `body` så kryptisk, som et tillegg til meldingen? Gir i det minste en indikasjon på hvor feilen ligger. Tenkte det er bedre enn å "dumpe" JSON-versjonen fra Pydantic i `message` som i dag? Alternativt kun med `err['msg']`? I så fall prisgitt at meldingene fra Pydantic er greie nok.

Reply:
Nei, synes at den nye generelle feilmeldingen:

```py
f"{location}: {err['msg']}"
```
er bedre enn den som var:

```py
str(err)
```

Jeg tenkte bare på tilfellene med de tre "skreddersydde" feilmeldingene ("Unsupported Maskinporten environment", "Invalid client ID" og "Invalid team ID"), der synes jeg at den `path`-prefiksgreia gjorde det mer kryptisk. Det burde jo være tydelig for brukeren hva som er galt ut fra den håndskrevne meldinga? Jeg skjønte i hvert fall ikke umiddelbart hva `path.client_id` refererte til. :sweat_smile:

Reply:
Ja, men hvis man har flere valideringsfeil i f.eks. `ClientIn`, hvorav noen ikke "fanges" av egendefinerte, så joines jo disse i dag sammen med en newline. Er det ikke da rart å ikke ha med tilsvarende "location" der? Kan selvsagt droppe det, men tror kanskje ikke alle `err['msg']` sier hvilket felt det er snakk om.